fpga/Makefile

```diff
@@ -120,6 +137,84 @@ debug-bitstream: $(build_dir)/obj/post_synth.dcp
 			$(build_dir)/debug_obj \
 			$(fpga_common_script_dir)
 
+#########################################################################################
+# sim rules
+#########################################################################################
+
+# hack until able to restructure fpga-shells .tcl scripts
+project-creation-script := $(fpga_common_script_dir)/create_project.tcl
+$(project-creation-script): $(fpga_common_script_dir)/vivado.tcl
+	cd $(fpga_common_script_dir); sed -n '1,10p' vivado.tcl > create_project.tcl
```

Comment:
I'm inclined to say that this should just reside in the `generated-src/$LONG_NAME` folder when created (and thus gets deleted when that is deleted). Then you don't need to re-generate this everytime a new sim is done.

Reply:
I agree. I ended up structuring this a bit differently in the most recent commit, creating a permanent `fpga/scripts/create_project.tcl` script, so no need for a sed-hacked temporary script living in fpga-shells.